### PR TITLE
add an introduction for testing of ECN support

### LINKED ISSUE #4109: Why do we talk about ECN validation in the chapter "Cryptographic and Transport Handshake"?
The sentence
> An endpoint verifies support for Explicit Congestion Notification (ECN) by
observing whether the ACK frames acknowledging the first packets it sends carry
ECN counts, as described in {{ecn-validation}}.

seems to not belong into that section.

### PR DESCRIPTION
**Pull request by marten-seemann**
Fixes #4109.

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -1463,9 +1463,10 @@ properties:
 * authenticated negotiation of an application protocol (TLS uses ALPN
   {{?ALPN}} for this purpose)
 
-An endpoint verifies support for Explicit Congestion Notification (ECN) by
-observing whether the ACK frames acknowledging the first packets it sends carry
-ECN counts, as described in {{ecn-validation}}.
+Endpoints can use packets sent during the handshake to test for Explicit
+Congestion Notification (ECN) ({{ecn}}) support. An endpoint verifies support
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:1467]:
```suggestion
Congestion Notification (ECN) support; see {{ecn}}. An endpoint verifies support
```

**Review by martinthomson — COMMENTED** [review]
WFM

**Review by ianswett — APPROVED** [review]

**Comment by ianswett** [comment]
Yeah, this is better.

**Review by janaiyengar — APPROVED** [review]
This is good, thanks!

**Pull request merged**